ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -5,125 +5,74 @@
-    }
-
-    let positionController: TooltipPositionController
-
-    private let message: String
-
-    private let backgroundView: UIView = {
-        let view = UIView()
-        view.layer.cornerRadius = backgroundCornerRadius
-        view.layer.cornerCurve = .continuous
-
-        return view
-    }()
-
-    private let arrowImageViewBaseImage: UIImage?
-    private let arrowImageView: UIImageView
-
-    private let messageLabel: UILabel = {
-        let label = Label(style: Constants.messageLabelTextStyle)
-        label.textColor = Colors.Tooltip.text
-        label.numberOfLines = 0
-        return label
-    }()
-
-    init(message: String, textAlignment: NSTextAlignment, positionController: TooltipPositionController) {
+    init(anchorView: UIView,
+         message: String,
+         title: String? = nil,
+         textAlignment: NSTextAlignment,
+         preferredArrowDirection: Tooltip.ArrowDirection,
+         offset: CGPoint,
+         arrowMargin: CGFloat,
+         tokenSet: TooltipTokenSet) {
+        self.anchorView = anchorView
         self.message = message
-        self.positionController = positionController
+        self.titleMessage = title
+        self.preferredArrowDirection = preferredArrowDirection
+        self.offset = offset
+        self.arrowMargin = arrowMargin
+        self.tokenSet = tokenSet
 
-        arrowImageViewBaseImage = UIImage.staticImageNamed("tooltip-arrow")
+        let arrowImageViewBaseImage = UIImage.staticImageNamed("tooltip-arrow")
         arrowImageView = UIImageView(image: arrowImageViewBaseImage)
+        arrowImageView.image = arrowImageViewBaseImage?.withTintColor(UIColor(dynamicColor: tokenSet[.tooltipColor].dynamicColor), renderingMode: .alwaysOriginal)
 
         super.init(frame: .zero)
 
-        isAccessibilityElement = true
-
         addSubview(backgroundView)
+        accessibilityViewIsModal = true
 
         arrowImageView.transform = transformForArrowImageView()
         addSubview(arrowImageView)
 
         messageLabel.text = message
         messageLabel.textAlignment = textAlignment
-        messageLabel.isAccessibilityElement = false
-        addSubview(messageLabel)
 
+        if let titleLabel = titleLabel, let title = title {
+            titleLabel.text = title
+            titleLabel.textAlignment = textAlignment
+        }
+
+        addSubview(textContainer)
+
+        // TODO: Integrate with new applyShadow functionality
         // Shadow
-        layer.shadowOpacity = Constants.shadowOpacity
-        layer.shadowRadius = Constants.shadowRadius
-        layer.shadowOffset = CGSize(width: 0.0, height: Constants.shadowOffset)
-    }
+        layer.insertSublayer(CALayer(), at: 0)
+        layer.insertSublayer(CALayer(), at: 0)
+        updateShadows()
 
-    required init?(coder aDecoder: NSCoder) {
-        preconditionFailure("init(coder:) has not been implemented")
+        isAccessibilityElement = true
```

Comment:
You might need to add the tokenSet sink closure to call token updates whenever the tokenSet is changed:

```suggestion
      // Update appearance whenever `tokenSet` changes.
       tokenSetSink = tokenSet.sinkChanges { [weak self] in
            // add token update calls here
        }
```
(Don't delete the `isAccessibility` line, I just don't know the proper markdown for adding multi-line code snippets)